src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/FundingCycleListItem.tsx

```diff
@@ -2,20 +2,46 @@ import { Tooltip } from 'antd'
 import { ThemeContext } from 'contexts/themeContext'
 import { CSSProperties, useContext } from 'react'
 
-function ListItemValue({ value }: { value: string | JSX.Element }) {
+type DiffStatus = 'new' | 'old'
+
+function ListItemValue({
+  value,
+  diffStatus,
+}: {
+  value: string | JSX.Element
+  diffStatus?: DiffStatus
+}) {
   const {
     theme: { colors },
   } = useContext(ThemeContext)
+  const highlight =
+    diffStatus === 'new'
+      ? `#AF${colors.background.failure.slice(1)}` // 69% opacity
+      : diffStatus === 'old'
+      ? `#AF${colors.background.success.slice(1)}`
+      : undefined
```

Comment:
@johnnyd-eth now that we have tailwind, can you pls update these colors to use Tailwind colors?